### PR TITLE
Translation enhancement for challenge Exact Change

### PR DESCRIPTION
**Pull request by S1ngS1ng**
## In this PR:
- Enhancement of translation. As per discussion in the issue, add details of `cid` array

### CODE REVIEW

**Review by wudifeixue — CHANGES_REQUESTED** [review]
They are pretty good so far, though please translate the comment example as well either in the comments or in the description. The name penny, nickel, dime, and quarter is really confusing for most people in China.

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 1]
```diff
@@ -177,7 +177,8 @@
       "description": [
         "设计一个收银机找零函数 <code>checkCashRegister()</code>，它接收三个参数，分别是购买总价 (<code>price</code>)、付款金额 (<code>cash</code>) 和收银机中可用的零钱 (<code>cid</code>)。",
         "<code>cid</code> 是一个二维数组，存着当前可用的找零。每个子数组中的第一个元素代表币种，第二个元素代表这个币种的可用总额。比如，<code>[\"FIVE\", 20.00]</code> 表示五元纸币的可用总额是 <code>20</code>，也就是有四张五元纸币。",
-        "当收银机中的钱不够找零时返回字符串 <code>\"Insufficient Funds\"</code>. 如果正好则返回字符串 <code>\"Closed\"</code>。",
+        "其中，<code>\"PENNY\"</code> 表示一分 (0.01)，<code>\"NICKLE\"</code> 表示五分 (0.05)，<code>\"DIME\"</code> 表示一角 (0.1)，<code>\"QUARTER\"</code> 表示两毛五 (0.25)。<code>\"ONE\"</code> 表示一元，<code>\"FIVE\"</code> 表示五元，其余的以此类推。",
```
Comment (S1ngS1ng) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: @wudifeixue I'd rather leave it in description 😂

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 2]
```diff
@@ -187,7 +188,7 @@
         "  return change;",
         "}",
         "",
-        "// Example cash-in-drawer array:",
+        "// cid 参数示例:",
```
Comment (S1ngS1ng) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json:191]: @wudifeixue Good enough?

Comment (wudifeixue) [reply seed/challenges/01-front-end-development-certification/advanced-bonfires.json:191]: Yup exactly what I want to know

**Review by wudifeixue — APPROVED** [review]

**Pull request merged**